src/presentation/pages/Note.vue

```diff
@@ -54,6 +70,41 @@ const { note, noteTools, save, noteTitle, canEdit } = useNote({
   id: noteId,
 });
 
+const lastEdit = ref<string>('Last edit ');
+
+/**
+ * Create new child note
+ */
+function createChildNote(): void {
+  if (props.id === null) {
+    throw new Error('Note is Empty');
+  }
+  router.push(`/note/${props.id}/new`);
+}
+
+/**
+ * Move to note settings
+ */
+function redirectToNoteSettings(): void {
+  if (props.id === null) {
+    throw new Error('Note is Empty');
+  }
+  router.push(`/note/${props.id}/settings`);
+}
+
+/**
+ * Loads the time when the note was last modified when opened
+ */
+watch(note, () => {
+  if (note.value !== null && 'updatedAt' in note.value) {
+    const updatedAt = note.value.updatedAt;
+
+    if (typeof updatedAt === 'string') {
+      lastEdit.value = 'Last edit ' + formatShortDate(updatedAt);
```

Comment:
Can we use i18n here? Or we dont have this ability rn

Reply:
I think we can, i'll do it